src/wallet/wallet.cpp

```diff
@@ -3975,14 +3985,6 @@ std::shared_ptr<CWallet> CWallet::CreateWalletFromFile(interfaces::Chain& chain,
     // Register with the validation interface. It's ok to do this after rescan since we're still holding locked_chain.
```

Comment:
This comment is no longer true, and seems like it means there can now be missing notifications. I wonder if a different approach would be to send the loadwallet notification before locking the chain & scanning instead of after?